cartridges/plugin_pd_image_editor/cartridge/templates/default/experience/components/commerce_assets/customHeroImage.isml

```diff
@@ -0,0 +1,14 @@
+<isscript>
+    var assets = require('*/cartridge/scripts/assets.js');
+    assets.addJs('/js/imageHelpersScripst.js');
+</isscript>
+
+<div>
+    <img class="img-pins" src="${pdict.imgUrl}" alt="${pdict.imageAlt}">
```

Comment:
Please add an option if this image to be loaded lazy or not.

Reply:
added lazy loading